src/normalize/entities.ts

```diff
@@ -263,3 +263,69 @@ export const SearchResponse = new Entity({
 	c: ['conversations', Conversation],
 	cn: ['contacts', Contact]
 });
+
+const RedirectAction = new Entity({
+	a: 'address'
+});
+
+const NotifyAction = new Entity({
+	a: 'address',
+	su: 'subject'
+});
+
+const FilterAction = new Entity({
+	actionKeep: 'keep',
+	actionDiscard: 'discard',
+	actionFileInto: 'fileInto',
+	actionFlag: 'flag',
+	actionTag: 'tag',
+	actionRedirect: ['redirect', RedirectAction],
+	actionReply: 'reply',
+	actionNotify: ['notify', NotifyAction],
+	actionStop: 'stop'
+});
+
+const DateCondition = new Entity({
+	d: 'date'
+});
+
+const ImportanceCondition = new Entity({
+	imp: 'importance'
+});
+
+const SizeCondition = new Entity({
+	s: 'size'
+});
+
+const FilterCondition = new Entity({
+	condition: 'allOrAny',
+
+	addressBookTest: 'addressBook',
```

Comment:
What was the importance of removing "Test" from the name of each filter condition?

Reply:
it's parent node is already indicating that it's a condition, so removed it from each level